README.md

```diff
@@ -55,6 +55,13 @@ To match emoji in their textual representation as well (i.e. emoji that are not
 const emojiRegex = require('emoji-regex/text.js');
 ```
 
+Additionally, in environments which support ES6 Unicode escapes, you may `require` ES6-style versions of the regexes:
+
+```js
+const emojiRegex = require('emoji-regex/unicode/index.js');
```

Comment:
How would you feel about using `/es2015/` instead of `/unicode/` (kind of vague in this context IMHO)?